cwf/src/lib.rs

```diff
@@ -34,7 +39,16 @@ fn rule_quote(rule: &Rule) -> TokenStream2 {
 		},
 		"href" => {
 			quote! {
-				current_element.set_attribute("href", #value)?;
+			    if #value.contains("."){
```

Comment:
this should all happen outside of the quote I think, the stuff in quote will happen at runtime but I don't see why not do it at compile time